mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.EraseOrder = async (req, res) => {
```

Comment:
The function `EraseOrder` is defined twice in the codebase, which can lead to conflicts and unexpected behavior. It should be removed from this file to avoid duplication.